ci/pipeline.yml

```diff
@@ -405,7 +405,7 @@ jobs:
                 type: docker-image
                 source:
                   repository: iacbox.common.repositories.cloud.sap/haproxy-boshrelease-testflight
-                  tag: "2022-08-03"
+                  tag: latest
```

Comment:
this could be dangerous. since we don't build the image regularly and push it to the registry, it will grow stale over time and we won't know what "latest" means when we look at it. we can't compare it to the actual latest from upstream then.
I'd rather opt for
a) keep the tag but bump it
b) keep "latest" but rebuild and push the image regularly

Reply:
I'm for b) and this is what I started now. `latest` is now in fact a fresh image (making sure it actually works properly at all)

My longer-term plan is to create a new concourse job that would create a new image when there are e.g. new stem cells and possibly once a week or something like that. This is ideally in combination with a 'canned director state' that could just be revived instead of building it up from scratch on every run.

Reply:
Agreed.